build-logic/plugins/src/main/kotlin/AndroidApplicationConventionPlugin.kt

```diff
@@ -38,8 +38,8 @@ class AndroidApplicationConventionPlugin : Plugin<Project> {
                 applicationId = AndroidApp.id
                 defaultConfig.targetSdk = AndroidSdk.target
                 versionCode = AndroidApp.versionCode
-                versionName = "${AndroidApp.versionName}($versionCode)"
-                setProperty("archivesBaseName", "$applicationId-v$versionName($versionCode)")
+                versionName = "${AndroidApp.versionName}_$versionCode"
+                setProperty("archivesBaseName", "$applicationId-v$versionName")
```

Comment:
We gotta check if there could be any issue with automation or naming we get for the builds with QA, just in case it doesn't break

Reply:
I reverted that change for now, we can always make it later when we're sure that it won't break automation